packages/notifications/src/Notification.php

```diff
@@ -108,14 +109,18 @@ public function broadcast(Model | Authenticatable | Collection | array $users):
         return $this;
     }
 
-    public function sendToDatabase(Model | Authenticatable | Collection | array $users): static
+    public function sendToDatabase(Model | Authenticatable | Collection | array $users, bool isEventDispatched = false): static
```

Comment:
Uhm, this might be why the tests are failing? 😂

```suggestion
    public function sendToDatabase(Model | Authenticatable | Collection | array $users, bool $isEventDispatched = false): static
```